#!/usr/bin/env python3
PACKAGE = "livox_camera_calib"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("grey_threshold", int_t, 0, "Canny grey threshold", 20, 1, 100)
gen.add("len_threshold", int_t, 0, "Canny length threshold", 200, 0, 1000)
gen.add("voxel_size", double_t, 0, "Point cloud voxel size", 1.0, 0.1, 3.0)
gen.add("execuate", bool_t, 0, "Execuate", False)

exit(gen.generate(PACKAGE, "livox_camera_calib", "CalibTune"))

